src/pages/BitcoinDashboardV2/index.tsx

```
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Header } from './components/Header';
+import Content from 'layouts/Content';
+import styles from './index.module.scss';
+import Checkpoint from './components/Checkpoint/Checkpoint';
+import { PendingDeposits } from './components/PendingDeposits';
+import { Tabs, KeysFilter } from './components/Tabs/Tabs';
+import { useSearchParams } from 'react-router-dom';
+import { PendingWithdraws } from './components/PendingWithdraws';
+import ConvertBitcoinV2 from './components/ConvertBitcoinV2';
+
+const BitcoinDashboard: React.FC<{}> = () => {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)